packages/cli/test/unit/commands/list.test.ts

```diff
@@ -69,67 +69,13 @@ describe('list', () => {
     data.shift();
     expect(data).toEqual([
       `https://${deployment.url}`,
-      stateString(deployment.state || ''),
+      stateString(deployment.readyState || ''),
       deployment.target === 'production' ? 'Production' : 'Preview',
       getDeploymentDuration(deployment),
       user.username,
     ]);
   });
 
-  it('should get deployments for linked project where the scope is a user', async () => {
```

Comment:
This test was removed because all users are northstar users now, so this was testing a case which can no longer occur.